easybuild/easyblocks/c/cuda.py

```diff
@@ -31,6 +36,14 @@ class EB_CUDA(Binary):
     Support for installing CUDA.
     """
 
+    @staticmethod
+    def extra_options():
+        extra_vars = {
+            'generate_intel_wrapper': [False, "Generate an nvcc wrapper (called invcc) to enable the usage of the Intel compiler as a host compiler, without explicitely using -ccbin.", CUSTOM],
+            'generate_gcc_wrapper': [False, "Generate an nvcc wrapper (called gnvcc) to enable the usage of the GCC compiler as a host compiler, without explicitely using -ccbin. g++ is the default host compiler. The wrapper is done just for clarity and completion (invcc for Intel and gnvcc for GCC)", CUSTOM],
```

Comment:
lines longer than 120 characters should be avoided, can you refactor this to split it across multiple lines (let me know if you need help on this)